spec.html

```diff
@@ -107,6 +107,30 @@ <h1>Well-Known Intrinsic Objects</h1>
 <emu-clause id="sec-abstract-operations">
   <h1>Abstract Operations</h1>
 
+  <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+    <h1>
+      SetterThatIgnoresPrototypeProperties (
+        _home_: an Object,
+        _p_: a property key,
+        _v_: an ECMAScript language value,
+      ): either a normal completion containing ~unused~ or a throw completion
+    </h1>
+    <dl class="header">
+    </dl>
+    <emu-alg>
+      1. Let _O_ be ? RequireObjectCoercible(*this* value).
```

Comment:
The next lines seems to be assume `O` is an object, so I guess this should actually be `ToObject(thisValue)` instead of `RequireObjectCoercible`.

Reply:
https://github.com/tc39/proposal-iterator-helpers/pull/290